libs/dextool/source/dextool/compilation_db/package.d

```diff
@@ -701,6 +703,17 @@ ParseFlags parseFlag(CompileCommand cmd, const CompileCommandFilter flag_filter,
         return 0 != flag.among("-D");
     }
 
+    /// Flags that is includes but contains spaces are wrapped in quotation marks.
+    static bool isSpacedInclude(string flag) @safe {
+	return flag[0 .. 2] == "-I" && indexOf(flag, '"') == 2;
```

Comment:
Have to check the length first before slicing. Otherwise if the string is ever less than 2 chars this will raise an exception.